### PR TITLE
Update Microsoft.DiaSymReader and Microsoft.DiaSymReader.Portable to their release builds

### PR DESCRIPTION
**Pull request by tmat**
**Customer scenario**

Updating pre-release dependencies to their respective release versions.

**Bugs this fixes:**

[VSO 507642](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/507642)

**Workarounds, if any**

None.

**Risk**

Small.

**Performance impact**

None.

**Is this a regression from a previous update?**

**Root cause analysis:**

Planned update.

**How was the bug found?**

N/A

**Test documentation updated?**

N/A

### CODE REVIEW

File: build/Targets/Packages.props [hunk 1]
```diff
@@ -35,11 +35,11 @@
     <MicrosoftCSharpVersion>4.3.0</MicrosoftCSharpVersion>
     <MicrosoftDiagnosticsRuntimeVersion>0.8.31-beta</MicrosoftDiagnosticsRuntimeVersion>
     <MicrosoftDiagnosticsTracingTraceEventVersion>1.0.35</MicrosoftDiagnosticsTracingTraceEventVersion>
-    <MicrosoftDiaSymReaderVersion>1.2.0-beta1-62008-01</MicrosoftDiaSymReaderVersion>
-    <MicrosoftDiaSymReaderConverterVersion>1.1.0-beta1-62106-03</MicrosoftDiaSymReaderConverterVersion>
-    <MicrosoftDiaSymReaderConverterXmlVersion>1.1.0-beta1-62106-03</MicrosoftDiaSymReaderConverterXmlVersion>
+    <MicrosoftDiaSymReaderVersion>1.2.0</MicrosoftDiaSymReaderVersion>
+    <MicrosoftDiaSymReaderConverterVersion>1.1.0-beta1-62217-01</MicrosoftDiaSymReaderConverterVersion>
```
Comment (tmat) [inline build/Targets/Packages.props]: Converter is only used in tests and thus can stay on pre-release bits for now.

**Comment by tmat** [comment]
test windows_release_unit64_prtest  please

**Comment by tmat** [comment]
test this please

**Comment by tmat** [comment]
failures due to nuget cache.

**Comment by tmat** [comment]
test windows_release_unit64_prtest please

**Comment by tmat** [comment]
Test failure: https://github.com/dotnet/roslyn/issues/19763

**Comment by tmat** [comment]
Test failure: https://github.com/dotnet/roslyn/issues/21559

**Comment by tmat** [comment]
test windows_release_vs-integration_prtest please

**Comment by tmat** [comment]
@MattGertz @MeiChin-Tsai For approval.
@natidea

**Comment by MattGertz** [comment]
We are in Escrow, so this needs a VSO bug with all of the appropriate decorations.  Set PU Triage to Submit when ready, and I'll triage tomorrow.

**Comment by MeiChin-Tsai** [comment]
Approved. thx.

**Comment by tmat** [comment]
@MattGertz  VSO 507642 is filled in.

**Comment by natidea** [comment]
Approved to merge via [507642](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/507642)

**Pull request merged**